README.md

```diff
@@ -1,264 +1,505 @@
+1. Within the extracted folder run:
 
 ```bash
-npm install # or `yarn install` if you want 3x the speed
+npm install
 npm run setup
 npm start
 ```
 
-If you're using Drupal 8, get the [Component Libraries module](https://www.drupal.org/project/components):
+Simply wait until the webpack bundle output appears then visit [http://0.0.0.0/pl](http://0.0.0.0/pl) (or [http://localhost/pl](http://localhost/pl)) and start working.
+
+### Quickstart with Drupal 8
+
+NAMETBD provides a Drupal 8 theme, the starting steps are slightly different:
+
+1. [Download the latest release](https://github.com/phase2/pattern-lab-starter/releases)
+1. Extract to `themes/` at the root of your Drupal 8 install. (i.e. this readme should be at `drupal-root/themes/NAMETBD/README.md`)
+1. Download and install the [Component Libraries module](https://www.drupal.org/project/components):
 
 ```bash
 drush dl components
 drush en components -y
 ```
 
-That's it.
+1. Within `drupal-root/themes/NAMETBD/` run:
+
+```bash
+npm install
+npm run setup
+npm run compile:drupal
+```
+
+This will compile all assets and provide all namespaces to the Drupal theme. Make sure to choose this theme in Drupal Appearance settings and `drush cr` to clear cache.
+
+For subsequent recompile and Drupal cache clear, run:
+
+```bash
+npm run compile:drupal && drush cr
+```
+
+Working rapidly in Pattern Lab is still available, simply run:
+
+```bash
+npm start
+```
+
+Just like in the regular Quickstart, when Webpack output appears, visit [http://0.0.0.0/pl](http://0.0.0.0/pl) (or [http://localhost/pl](http://localhost/pl)) to immediately start building and previewing your design system in Pattern Lab.
+
+That's it. For **much** greater detail on the frontend approach using this project, check out the [Phase2 Frontend Docs](https://phase2.github.io/frontend-docs/).
 
 ## Commands
 
-Compile everything:
+Quick compile Pattern Lab:
 
 ```bash
-npm run compile
+npm run compile:pl
 ```
 
-Start up watches and local server after compiling:
+Quick compile Drupal
 
 ```bash
-npm run start # or `npm start`
+npm run compile:drupal
 ```
 
-> Protip: any config option from `gulpconfig.yml` can be overwritten with `npm start -- --js.enabled=''`, or by including options in your own `~/.p2-theme-corerc` file. See [`rc`](https://www.npmjs.com/package/rc) for more details.
+Start up watches and local server:
+
+```bash
+npm start
+```
 
-Run Tests:
+Run all linters:
 
 ```bash
-npm run test # or `npm test`
+npm run lint
 ```
 
-Create a new component folder in Pattern Lab with scss, twig, md, & yml/json by running:
+Run all tests:
 
 ```bash
-npm run new
+npm test
 ```
 
-To update node and composer dependencies (if you see messages about Pattern Lab wanting to Merge or Replace files, merge them):
+Run accessibility testing on Pattern Lab rendered output:
 
 ```bash
-npm run update
+npm run test:accessibility
 ```
 
-## Assets
+To update node and composer dependencies (**merge** if offered the option):
```

Comment:
"merge" note lacks any context here. The removed text was a bit better, but also vague. Are you referring to Pattern Lab wanting to merge a composer dependency?

Reply:
Composer updating Pattern Lab occasionally offers to "merge" new file structures to existing file structures. Doesn't feel like this is the place to get into it.